express-get-json/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.use('/api/grades', (req, res) => {
+  res.json(grades);
+});
```

Comment:
This should send back an array of grade objects. You need to use a `for in` loop to build the array, then send the array back to the client.

Reply:
![Screen Shot 2022-08-31 at 11 12 05 PM](https://user-images.githubusercontent.com/44100814/187844443-4285a164-1c1e-4979-92e9-9f13d027c2d3.png)